crates/turborepo-paths/src/relative_unix_path.rs

```diff
@@ -19,7 +19,7 @@ impl RelativeUnixPath {
         Ok(unsafe { &*(path as *const BStr as *const Self) })
     }
 
-    pub(crate) fn to_system_path_buf(&self) -> Result<PathBuf, PathError> {
+    pub fn to_system_path_buf(&self) -> Result<PathBuf, PathError> {
```

Comment:
Where are we using this?

Reply:
Good catch. An earlier version was using it.